chart/values.yaml

```
@@ -207,7 +207,7 @@ persistence:
     # -- Node selector for the default Longhorn StorageClass. Longhorn uses only nodes with the specified tags for storing volume data. (Examples: "storage,fast")
     selector: ""
   # -- Setting that allows you to enable automatic snapshot removal during filesystem trim for a Longhorn StorageClass. (Options: "ignored", "enabled", "disabled")
-  removeSnapshotsDuringFilesystemTrim: ignored
+  removeSnapshotsDuringFilesystemTrim: "false"
```

```suggestion
  unmapMarkSnapChainRemoved: ignored
```
Here are the `persistence` settings for the StorageClass as you modified in the `templates/storagclass.yaml`.

We also need to modify the README.md and questions.yaml.

---

Good catch! Fixed. Thank you.